### ISSUE TITLE
Bug: Issue viewing pods from Argo Rollouts

### ISSUE THREAD

**Issue by patrick0057** [rancher/rancher#27923]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
1) Deploy a replica set with the following YAML
```
apiVersion: apps/v1
kind: ReplicaSet
metadata:
  annotations:
    rollout.argoproj.io/desired-replicas: "3"
    rollout.argoproj.io/revision: "3"
  creationTimestamp: "2020-07-08T22:23:25Z"
  generation: 5
  labels:
    app: argo-demo-rollouts
    rollouts-pod-template-hash: 6b4d44785c
  name: argo-demo-rollouts-6b4d44785c
  namespace: default
  resourceVersion: "81506683"
  selfLink: /apis/extensions/v1beta1/namespaces/default/replicasets/argo-demo-rollouts-6b4d44785c
  uid: 094e0074-635d-4a0f-8ffd-1157e776ddf4
spec:
  replicas: 3
  selector:
    matchLabels:
      app: argo-demo-rollouts
      rollouts-pod-template-hash: 6b4d44785c
  template:
    metadata:
      creationTimestamp: null
      labels:
        app: argo-demo-rollouts
        rollouts-pod-template-hash: 6b4d44785c
    spec:
      containers:
      - image: ubuntu
        imagePullPolicy: IfNotPresent
        name: argo-demo-rollouts
        ports:
        - containerPort: 80
          protocol: TCP
        resources: {}
        terminationMessagePath: /dev/termination-log
        terminationMessagePolicy: File
      dnsPolicy: ClusterFirst
      restartPolicy: Always
      schedulerName: default-scheduler
      securityContext: {}
      terminationGracePeriodSeconds: 30
```

**Result:**
In the workloads section, you cannot see the pods when you click on the Replica set.

**Other details that may be helpful:**
issue is similar to https://github.com/rancher/rancher/issues/22106

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): v2.4.5
- Installation option (single install/HA): HA

gzrancher/rancher#11166

**Comment by maggieliu** [comment]
All pods should be available in the 2.5 Cluster Explorer view.

**Comment by izaac** [comment]
Rancher version: `master-head (09/17/2020)` _33f755f_

All pods appear in the v2.5 Cluster Explorer view.
Checked with the provided yaml and with the Getting Started Guide

### Ember UI

![Screen Shot 2020-09-17 at 11 28 03 AM](https://user-images.githubusercontent.com/27659/93512250-e5306580-f8d8-11ea-965e-0541cbe8fdb6.png)

### Cluster Explorer views

![Screen Shot 2020-09-17 at 11 27 52 AM](https://user-images.githubusercontent.com/27659/93512302-fbd6bc80-f8d8-11ea-8165-2b65f901bf0c.png)


![Screen Shot 2020-09-17 at 11 27 45 AM](https://user-images.githubusercontent.com/27659/93512330-02fdca80-f8d9-11ea-9921-d234ae0d4314.png)


![Screen Shot 2020-09-17 at 11 27 34 AM](https://user-images.githubusercontent.com/27659/93512349-0c873280-f8d9-11ea-851f-e8bfc82b8dda.png)
